ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -9,13 +9,15 @@
   - ISIS-LDP*.
   - ISIS-SR-LDP*.
   - OSPF-LDP*.
+  - none (for use with l2ls)**.
 - The following overlay routing protocols are supported:
   - EBGP (default for l3ls-evpn)
   - IBGP (only with OSPF or ISIS variants in underlay)
 - Only summary network addresses need to be defined. IP addresses are then assigned to each node, based on its unique device id.
   - To view IP address allocation and consumption, a summary is provided in the auto-generated fabric documentation in Markdown and CSV format.
 
 *Only supported with core_interfaces data model.
+** For use with design type l2ls
```

Comment:
In Github UI these two lines do not render on separate lines. Cf https://gist.github.com/shaunlebron/746476e6e7a4d698b373 to check.

Otherwise could maybe use footnotes https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/ but would need to check that they make it to the mkdocs generated documentation on avd.sh (in theory mkdocs extension support the same syntax: https://squidfunk.github.io/mkdocs-material/reference/footnotes/#configuration as long asthe extension is enabled)

Reply:
resolved by adding newline.